lib/data_migrate/rails_helper.rb

```diff
@@ -54,7 +63,9 @@ def data_schema_delete_version(version)
       end
 
       def data_schema_migration
-        if rails_version_equal_to_or_higher_than_7_1
+        if rails_version_equal_to_or_higher_than_7_2
+          DataMigrate::DataSchemaMigration.new(ActiveRecord::Tasks::DatabaseTasks.migration_connection.pool)
```

Comment:
Base on [Rails' code](https://github.com/rails/rails/blob/main/activerecord/lib/active_record/migration.rb#L186), would it be fair to use `#migration_connection_pool` instead?

Reply:
Good call, turns out ActiveRecord::Base also has `connection_pool` over `connection.pool` 👍